public/config/configuration.json

```diff
@@ -1,11 +1,11 @@
 {
-  "BUSINESSES_URL": "https://dev.bcregistry.ca/cooperatives/auth/",
-  "AUTH_URL": "https://dev.bcregistry.ca/cooperatives/auth/",
+  "BUSINESSES_URL": "https://dev.bcregistry.ca/business/auth/",
+  "AUTH_URL": "https://dev.bcregistry.ca/business/auth/",
```

Comment:
In case anyone is wondering... while these have the same value atm, they are used for different purposes and I expect them to different in the near future (after MVP).

Reply:
ah okay I was wondering that too. Thank you for clarifying